doc/sphinx/source/notes_6_0.rst

```diff
@@ -20,6 +20,15 @@ any particular behavior (or lack thereof) they are treated as a no-op. However
 specifying them no longer causes `clCreateCommandQueueWithProperties` to return
 an error.
 
+============================
+Multi-device command buffers
+============================
+
+Initial support for `cl_khr_command_buffer_multi_device` has been added, i.e. it
```

Comment:
For the CPU devices, right?

Reply:
There is not really anything driver-specific, just like in the original command buffer code. It just hasn't really been tested with other drivers.

Reply:
Perhaps still worth mentioning what are the expectations in terms of perf. improvements (none still, right?) as it "caches" mainly the error checking?